diff --git a/configure/ProjectFile.cpp b/configure/ProjectFile.cpp
index a46248a..7941bfa 100644
--- a/configure/ProjectFile.cpp
+++ b/configure/ProjectFile.cpp
@@ -621,7 +621,7 @@ void ProjectFile::writeVS2010(wofstream &file,const vector<Project*> &allProject
   else if (_wizard->visualStudioVersion() == VS2013)
     file << "    <PlatformToolset>v120_xp</PlatformToolset>" << endl;
   else if (_wizard->visualStudioVersion() == VS2015)
-    file << "    <PlatformToolset>v140_xp</PlatformToolset>" << endl;
+    file << "    <PlatformToolset>v140</PlatformToolset>" << endl;
   else if (_wizard->visualStudioVersion() == VS2017)
     file << "    <PlatformToolset>v141_xp</PlatformToolset>" << endl;
   file << "    <UseOfMfc>false</UseOfMfc>" << endl;
